api/server/httputils/form.go

```diff
@@ -49,6 +49,16 @@ type ArchiveOptions struct {
 	Path string
 }
 
+type validationError struct {
+	Err error
+}
+
+func (e validationError) Error() string {
+	return e.Err.Error()
```

Comment:
Let's generate the error message here instead of wrapping a textual message. We could call this `badParameterError`.